app/javascript/controllers/search_bar_controller.js

```
@@ -201,4 +265,227 @@ export default class extends Controller {
     }
+
+    // =========================================================================
+    // Suggestions / Autocomplete (Issue #7340)
+    // =========================================================================
+
+    /**
+     * Returns the last word (token) being typed in the input, i.e. text after
+     * the last space character.  Used to filter suggestions contextually.
+     */
+    getLastWord() {
+        const value = this.inputTarget.value;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)